readme.md

```diff
@@ -116,7 +138,8 @@ How is rebase different than a merge?
 
 Your Answer:
 ```text
-
+A rebase will alter the commit history as if the changes to the branch being
+merged in had been there all along.
```

Comment:
Pretty close, a rebase is applying the changes between a branch and its root on top of the head of another branch